operator/operator.go

```diff
@@ -41,41 +41,42 @@ const AVS_NAME = "incredible-squaring"
 const SEM_VER = "0.0.1"
 
 type Operator struct {
-	config    types.NodeConfig
-	logger    sdklogging.Logger
-	ethClient sdkcommon.EthClientInterface
+	Config    types.NodeConfig
+	Logger    sdklogging.Logger
+	EthClient sdkcommon.EthClientInterface
 	// TODO(samlaf): remove both avsWriter and eigenlayerWrite from operator
 	// they are only used for registration, so we should make a special registration package
 	// this way, auditing this operator code makes it obvious that operators don't need to
 	// write to the chain during the course of their normal operations
 	// writing to the chain should be done via the cli only
-	metricsReg       *prometheus.Registry
-	metrics          metrics.Metrics
-	nodeApi          *nodeapi.NodeApi
-	avsWriter        *chainio.AvsWriter
-	avsReader        chainio.AvsReaderer
-	avsSubscriber    chainio.AvsSubscriberer
-	eigenlayerReader sdkelcontracts.ChainReader
-	eigenlayerWriter sdkelcontracts.ChainWriter
-	blsKeypair       *bls.KeyPair
-	operatorId       sdktypes.OperatorId
-	operatorAddr     common.Address
+	MetricsReg       *prometheus.Registry
+	Metrics          metrics.Metrics
+	NodeApi          *nodeapi.NodeApi
+	AvsWriter        *chainio.AvsWriter
+	AvsReader        chainio.AvsReaderer
+	AvsSubscriber    chainio.AvsSubscriberer
+	EigenlayerReader sdkelcontracts.ChainReader
+	EigenlayerWriter sdkelcontracts.ChainWriter
+	BlsKeypair       *bls.KeyPair
+	OperatorId       sdktypes.OperatorId
+	OperatorAddr     common.Address
 	// receive new tasks in this chan (typically from listening to onchain event)
-	newTaskCreatedChan chan *cstaskmanager.ContractIncredibleSquaringTaskManagerNewTaskCreated
+	NewTaskCreatedChan chan *cstaskmanager.ContractIncredibleSquaringTaskManagerNewTaskCreated
```

Comment:
Why were these made public?

Reply:
used it here https://github.com/Layr-Labs/incredible-squaring-avs/blob/0bf35223cfb426c125af777a9e233f19f009a3ed/plugin/cmd/main.go#L254-L273

Reply:
ok moving these to a separate deployment script

Reply:
done